src/helpers.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Gets a keyword describing an HTTP status code
+ *
+ * @param {String} statusCode HTTP status code
+ * @return {String} `'success'` if starts with `'1'` or `'2'`, `'unmodified'` if equals `'304'`,
+ * `'failure'` if starts with `'4'`, `'error'` if starts with `'5'`, `'invalid_status'` otherwise
+ */
+
+export const getStatusType = statusCode => {
+  if (statusCode[0] === '1' || statusCode[0] === '2') {
+    return 'success'
+  } else if (statusCode === '304') {
+    return 'unmodified'
+  } else if (statusCode[0] === '3') {
+    return 'redirect'
+  } else if (statusCode[0] === '4') {
+    return 'failure'
+  } else if (statusCode[0] === '5') {
+    return 'error'
+  } else {
+    return 'invalid_status'
+  }
+}
+
+/**
+ * Gets the time in ms since the request started
+ *
+ * @return {Number} Time in ms since the request started
+ */
+
+export const getRequestDuration = startTime => new Date().getTime() - startTime
```

Comment:
`process#hrtime` gives you the diff in nanoseconds. We need the duration in ms, so we have to convert it.
That substraction gives you the diff in ms. I don't know if it is less precise, but for sure we don't have the (possible?) error in the conversion.